x-pack/solutions/security/plugins/cloud_security_posture/server/tasks/findings_stats_task.ts

```diff
@@ -39,10 +40,141 @@ import {
   type TaskHealthStatus,
 } from './task_state';
 import { toBenchmarkMappingFieldKey } from '../lib/mapping_field_util';
+import { benchmarkScoreMapping } from '../create_indices/benchmark_score_mapping';
+import { createBenchmarkScoreIndex } from '../create_indices/create_indices';
+import type { CloudSecurityPostureConfig } from '../config';
 
 const CSPM_FINDINGS_STATS_TASK_ID = 'cloud_security_posture-findings_stats';
 const CSPM_FINDINGS_STATS_TASK_TYPE = 'cloud_security_posture-stats_task';
 
+// Default config for template fixing operations
+const DEFAULT_CONFIG: CloudSecurityPostureConfig = {
+  enabled: true,
+  serverless: { enabled: true },
+  enableExperimental: [],
+};
+
+const fetchBenchmarkScoreTemplate = async (esClient: ElasticsearchClient) => {
+  const templateResponse = await esClient.indices.getIndexTemplate({
+    name: BENCHMARK_SCORE_INDEX_TEMPLATE_NAME,
+  });
+  return templateResponse.index_templates[0]?.index_template;
+};
+
+const verifyTemplateStructure = (template: any, logger: Logger): boolean => {
```

Comment:
Instead of any, would be nice if we can type it with properly, can you try typing with `IndicesPutIndexTemplateRequest`?

```tsx
import type { IndicesPutIndexTemplateRequest } from '@elastic/elasticsearch/lib/api/types';
```